src/site/content/en/blog/catalina-system-ui/index.md

```diff
@@ -0,0 +1,246 @@
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_wght.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_wght.mp4">
+  </video>
+</figure>
+
+### `opsz` {: #opsz }
+Optical sizing is similar to kerning or letter-spacing, but the spacing is done by a human eye instead of math. A value of `19` or below is intended for text and body copy spacing, while `20` or above is for spacing display headers and titles.
+
+```css
+/* 19 or 20 */
+font-variation-settings: 'opsz' 20;
+```
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_opsz_v2.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_opsz_v2.mp4">
+  </video>
+</figure>
+
+### `GRAD`
+Similar to weight, but without touching horizontal spacing. It accepts values between `400` and `1000`.
+
+```css
+/* 400-1000 */
+font-variation-settings: 'GRAD' 500;
+```
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_grad_v2.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_grad_v2.mp4">
+  </video>
+</figure>
+
+### `YAXS`
+Stretches the glyph vertically. It accepts values between `400` and `1000`.
+
+```css
+/* 400-1000 */
+font-variation-settings: 'YAXS' 500;
+```
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_yaxs_v2.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_yaxs_v2.mp4">
+  </video>
+</figure>
+
+### Combining the options
+With a few lines of CSS, we can tweak the font settings into a bold of our choice or try out other interesting combinations:
+
+```css
+font-weight: 700;
+font-weight: bold;
+font-variation-settings: 'wght' 750, 'YAXS' 600, 'GRAD' 500, 'opsz' 20;
+```
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_combined_v2.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_combined_v2.mp4">
+  </video>
+</figure>
+
+And just like that, Chromium users on macOS see your upgraded, custom 750 weight with some fun other tweaks 👍 
+
+## Sandbox
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/macos-system-ui?path=style.css&attributionHidden=true"
```

Comment:
```suggestion
    src="https://glitch.com/embed/#!/embed/macos-system-ui?path=style.css&attributionHidden=true&previewSize=50"
```
This should show the source code and live app at the same time so that readers don't have to switch back and forth

Reply:
P.S. I love the Glitch here!